src/utils.ts

```diff
@@ -0,0 +1,9 @@
+import * as core from '@actions/core';
+
+export const IS_WINDOWS = process.platform === 'win32';
+export const IS_LINUX = process.platform === 'linux';
+
+export function logWarning(message: string): void {
+  const warningPrefix = '[warning]';
+  core.info(`${warningPrefix}${message}`);
```

Comment:
```suggestion
  core.info(`${warningPrefix} ${message}`);
```

Do we want a space before the start of the message?  Not sure how other actions do it.

Reply:
Other actions don't have white space there, for instance, [setup-go](https://github.com/actions/setup-go/blob/30b9ddff1180797dbf0efc06837929f98bdf7af7/src/cache-save.ts#L76-L79), [setup-python](https://github.com/actions/setup-python/blob/a93d5412cbea8e722be8a13812f0923da907d4cf/src/utils.ts#L141-L144).

Reply:
Can we just use the [`warning()`](https://github.com/actions/toolkit/blob/64c334f0e54c45f0ee319b5707281d9df0c43a3d/packages/core/src/core.ts#L274) function from `@actions/core`?